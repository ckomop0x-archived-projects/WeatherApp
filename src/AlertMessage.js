import React from 'react';

export default function AlertMessage() {
	return <div>*Because of http source at api.openweathermap.org you could allow to load resources from unsecure places to let the WeatherApp work.</div>;
}
