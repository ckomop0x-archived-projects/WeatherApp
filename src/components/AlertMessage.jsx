import React from 'react';

const AlertMessage = () =>  {
    return <div>*Because of http source at api.openweathermap.org you could allow to load
        resources from unsecure places to let the WeatherApp work.</div>;
};

export default AlertMessage;
