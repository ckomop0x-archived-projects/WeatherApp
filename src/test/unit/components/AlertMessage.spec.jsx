import React from 'react';
import renderer from 'react-test-renderer';
import AlertMessage from '../../../components/AlertMessage';


describe('Components', () => {
    describe('App', () => {
        it('renders without crashing', () => {
            const wrapper = renderer.create(<AlertMessage/>).toJSON();
            expect(wrapper).toMatchSnapshot();
        });
    });
});