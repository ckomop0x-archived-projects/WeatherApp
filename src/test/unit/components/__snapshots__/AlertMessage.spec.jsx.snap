// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components App renders without crashing 1`] = `
<div>
  <sup>
    *
  </sup>
   Because of http source at api.openweathermap.org you could allow to load resources from unsecure places to let the WeatherApp work.
</div>
`;
